fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -38,12 +39,131 @@
 UNEXPECTED_DATA_TYPE_ERROR_MESSAGE = "Unexpected data type {} encountered."
 ESTIMATOR_ERROR_MESSAGE = "'estimator' cannot be `None`."
 
-_SUPPORTED_CONSTRAINTS = [DEMOGRAPHIC_PARITY, EQUALIZED_ODDS]
-
 logger = logging.getLogger(__name__)
 
+# All allowed utilities must have values between 0 and 1 and attain both extremes as
+# the threshold goes from -Inf to Inf.
+SUPPORTED_UTILITIES = {
+    'selection_rate',
```

Comment:
This is equivalent to DP. Why add it?

Reply:
Or rather: why add this one and not others like equality of opportunity and other duplicate names?

Reply:
I'd like to limit the duplication unless there is a strong reason against it. The "systematic" terminology is "selection_rate", but "demographic_parity" is extremely common, so it's added as a synonym (and for the backward compatibility).

Reply:
Sure, if you feel strongly then go with that. My question was rather out of curiosity.